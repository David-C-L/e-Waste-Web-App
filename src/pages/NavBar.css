.NavBar {
    text-align: center;
}

.NavBarLink {
    padding: 2ex;
}