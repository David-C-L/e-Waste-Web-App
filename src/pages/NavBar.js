import './NavBar.css';
import {Link} from 'react-router-dom'

function NavBar() {
  return (
    <div className='NavBar'>
      <p>
        <Link className='NavBarLink' to='/'>Home</Link>
        <Link className='NavBarLink' to='/market'>Marketplace</Link>
        <Link className='NavBarLink' to='/requests'>Requests</Link>
        <Link className='NavBarLink' to='/feed'>Repair Forum</Link>
        <Link className='NavBarLink' to='/login'>Login</Link>
      </p>
    </div>
  );
}

export default NavBar;