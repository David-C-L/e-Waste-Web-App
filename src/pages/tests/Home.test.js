import NavBar from '../NavBar';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';


describe('NavBar component', () => {
    it('matches the snapshot', () => {
        const tree = renderer.create(<BrowserRouter><NavBar /></BrowserRouter>).toJSON()

        expect(tree).toMatchSnapshot()
    })
})