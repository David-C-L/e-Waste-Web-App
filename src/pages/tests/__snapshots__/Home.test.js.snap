// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component matches the snapshot 1`] = `
<div
  className="NavBar"
>
  <p>
    <a
      className="NavBarLink"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="NavBarLink"
      href="/market"
      onClick={[Function]}
    >
      Marketplace
    </a>
    <a
      className="NavBarLink"
      href="/request"
      onClick={[Function]}
    >
      Requests
    </a>
    <a
      className="NavBarLink"
      href="/feed"
      onClick={[Function]}
    >
      Repair Forum
    </a>
    <a
      className="NavBarLink"
      href="/login"
      onClick={[Function]}
    >
      Login
    </a>
  </p>
</div>
`;
